Missing include

If you add `target_compile_definitions(yomm2 PUBLIC YOMM2_ENABLE_TRACE=0)` to top level CMakeLists.txt, and build the project's example, it will results

```
D:\yomm2\src\yomm2.cpp(858,42): error C2079: 'discard_log' uses undefined class 'std::basic_ostringstream<char,std::char_traits<char>,std::allocator<char>>' [D:\yomm2\build\src\yomm2.vcxproj]
D:\yomm2\src\yomm2.cpp(864,50): error C2446: ':': no conversion from 'std::ostringstream' to 'std::ostream' [D:\yomm2\build\src\yomm2.vcxproj]
```

Compiler:
```
Microsoft (R) C/C++ Optimizing Compiler Version 19.26.28806 for x64
Copyright (C) Microsoft Corporation.  All rights reserved.
```

--- a/src/yomm2.cpp
+++ b/src/yomm2.cpp
@@ -14,6 +14,7 @@
 #include <random>
 
 #if YOMM2_ENABLE_TRACE
+#include <sstream>
 #include <iostream>
 #endif
 
@@ -852,6 +853,7 @@ bool runtime::is_base(const rt_spec* a, const rt_spec* b)
     return result;
 }
 
+#if YOMM2_ENABLE_TRACE
 std::ostream* active_log = nullptr;
 
 std::ostream& log() {
@@ -875,6 +877,7 @@ std::ostream* log_off() {
     active_log = nullptr;
     return prev;
 }
+#endif
 
 void default_method_call_error_handler(const method_call_error& error) {
 #if YOMM2_ENABLE_TRACE

